CHANGELOG.md

```diff
@@ -13,7 +16,7 @@
 - Fix aligning comments of different group
 - Fix flattening imports with a single `self`.
 - Fix removing attributes on function parameters.
-- Fix removing `impl` keyword from opaque type. 
```

Comment:
My editor removed some trailing whitespace in this file. If I should remove those stray changes, just tell me.